modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -12,15 +12,15 @@ module Mobile
   module V0
     class ClaimsAndAppealsController < ApplicationController
       include IgnoreNotFound
-      before_action(only: %i[index get_claim]) do
+      before_action(only: %i[index get_claim request_decision]) do
         if Flipper.enabled?(:mobile_lighthouse_claims, @current_user)
           authorize :lighthouse, :access?
         else
           authorize :evss, :access?
         end
       end
 
-      before_action(except: %i[index get_claim]) do
+      before_action(except: %i[index get_claim request_decision]) do
```

Comment:
Why are we duplicating this? Shouldn't the method above be handling it?

Reply:
there's a few more custom actions that don't have a lighthouse integration. once everything in the controller is integrated, it will turn into just the top before_action.

Reply:
Oh, right. I kinda lost track of the only vs. except. Which is funny because I think I'm the one who suggested this to begin with.